test/cpp/jit/test_gpu.cpp

```diff
@@ -5125,6 +5125,221 @@ TEST(NVFuserTest, FusionBCastAfterReduce_CUDA) {
   TORCH_CHECK(t5.allclose(outputs[0], 1e-5, 1e-5));
 }
 
+TEST(NVFuserTest, FusionOutputBroadcast_CUDA) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  TensorView* tv0 = makeConcreteTensor({2, 3});
+  fusion.addInput(tv0);
+
+  TensorView* tv1 = broadcast(tv0, {true, false, true, false, true});
+
+  fusion.addOutput(tv1);
+
+  const auto options =
+      at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
+
+  at::Tensor input = at::randn({2, 3}, options);
+
+  FusionExecutor fe;
+  fe.compileFusion(&fusion);
+
+  auto outputs = fe.runFusion({input});
+  auto aten_output = input;
+
+  TORCH_CHECK(
+      aten_output.allclose(outputs[0].squeeze(), 1e-04, 1e-04),
+      "Error of: ",
+      aten_output.sub(outputs[0].squeeze()).abs().max());
+}
+
+TEST(NVFuserTest, FusionReductionKeepDimBasic_CUDA) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  TensorView* tv0 = makeConcreteTensor({2, 3, 4, 5, 6});
+  fusion.addInput(tv0);
+
+  TensorView* tv1 = sum(tv0, {0, 2, 4}, /*keep_dim=*/true);
+
+  fusion.addOutput(tv1);
+
+  const auto options =
+      at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
+
+  at::Tensor input = at::randn({2, 3, 4, 5, 6}, options);
+
+  FusionExecutor fe;
+  fe.compileFusion(&fusion);
+
+  auto outputs = fe.runFusion({input});
+  auto aten_output = input.sum({0, 2, 4});
```

Comment:
Can we use the keepdim = true here, so we can get rid of the squeeze below?

Reply:
Sure, updated all the test cases to remove the squeeze.